ui/src/components/DraggableDialog.tsx

```diff
@@ -52,7 +52,7 @@ function PaperComponent(props: PaperProps) {
 	);
 }
 
-interface DraggableDialogProps extends DialogProps {
+interface DraggableDialogProps extends Omit<DialogProps, "content"> {
```

Comment:
MUI updates cause conflict between a `content` property of `DialogProps` and field added below for this custom interface, so omit `content` field from the inherited interface